doc/man1/flux-kvs.adoc

```diff
@@ -77,7 +77,9 @@ If '-r', value is displayed without a newline.  If '-t', the RFC 11 object
 is displayed.  '-a treeobj' causes the lookup to be relative to an RFC 11
 snapshot reference.  If '-w', after the initial value, display the
 new value each time the key changes until interrupted, or if '-c count'
-is specified, until 'count' values have been displayed.
+is specified, until 'count' values have been displayed.  If the
+initial value does not yet exist, `-W` can be specified to wait for it
+to be crated.
```

Comment:
Sorry for not catching this earlier. (copy and pasted later on in the .adoc too)

crated -> created

Reply:
oops ... I'll stick that fix onto my next PR.